components/grid/grid.css

```diff
@@ -0,0 +1,55 @@
+.grid-two-equal-container {
+  width: 100%;
+  height: 100%;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-areas: "container1 container1 container2 container2";
+}
+.grid-three-equal-container {
+  width: 100%;
+  height: 100%;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr;
+  grid-template-areas: "container1 container1 container2 container2 container3 container3";
+}
+.grid-four-equal-container {
+  width: 100%;
+  height: 100%;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-areas: "container1 container2 container3 container4";
+}
+
+.grid-wrapper {
+  height: 14rem;
```

Comment:
always add `min-height` unless you are using overflow-scroll.

Reply:
I did'nt thought about it, this is very valuable suggestion, I will make changes